content/open-source/apko/getting-started-with-apko.md

```diff
@@ -120,17 +120,20 @@ docker run --rm -v ${PWD}:/work -w /work cgr.dev/chainguard/apko build wolfi-bas
 You should get output similar to this:
 
 ```
-Feb  8 17:21:33.665 [INFO] loading config file: wolfi-base.yaml
-Feb  8 17:21:33.666 [INFO] [arch:x86_64] WARNING: ignoring archs in config, only building for current arch (amd64)
-Feb  8 17:21:33.666 [INFO] [arch:x86_64] building image 'wolfi-base:test'
-...
-Feb  8 17:21:41.772 [INFO] [arch:x86_64] finished building filesystem in /tmp/apko-2993726922
-Feb  8 17:21:41.845 [INFO] [arch:x86_64] built image layer tarball as /tmp/apko-temp-3502723090/apko-x86_64.tar.gz
-&{ID:wolfi IDLike: Name:Wolfi PrettyName:Wolfi Version: VersionID:20230201 VersionCodename:}Feb  8 17:21:41.955 [INFO] [arch:x86_64] building OCI image from layer '/tmp/apko-temp-3502723090/apko-x86_64.tar.gz'
-Feb  8 17:21:42.064 [INFO] [arch:x86_64] OCI layer digest: sha256:8f62e40cd09e48c90e93a0119a45e49fd50e9cd4cf0561a618b708623ed106e1
-Feb  8 17:21:42.064 [INFO] [arch:x86_64] OCI layer diffID: sha256:bf6e72d71c134bb210fac46f8abd9218821412d8bf9219f48d1b5e9e243e8233
-Feb  8 17:21:42.064 [WARNING] [arch:x86_64] multiple SBOM formats requested, uploading SBOM with media type: spdx+json
-Feb  8 17:21:42.068 [INFO] [arch:x86_64] output OCI image file to wolfi-test.tar
+Mar 15 20:17:02.023 [INFO] loading config file: wolfi-base.yaml
```

Comment:
Do we need this big section of output? Can we condense?

Reply:
Makes sense, a lot of it isn't meaningful to the guide as written - updated!